shared-bindings/usb_cdc/Serial.c

```diff
@@ -273,9 +273,7 @@ STATIC const mp_stream_p_t usb_cdc_serial_stream_p = {
     .write = usb_cdc_serial_write_stream,
     .ioctl = usb_cdc_serial_ioctl_stream,
     .is_text = false,
-    .pyserial_read_compatibility = true,
```

Comment:
Consumers can't work with streams that don't implement the expected behavior. If compatibility with pyserial is a must, I recommend creating a wrapper class that wraps a stream and exposes the pyserial interface.